configmap/deployment.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cloudnatived
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cloudnatived
+    spec:
+      containers:
+        - name: cloudnatived
+          image: cloudnatived/cloudnatived:v5
```

Comment:
The image spec for the first iteration (`docker run`) is `cloudnatived/demo:hello`, so how about `cloudnatived/demo:hello-config` for this one? (Can you have hyphens in tags?)